### PR TITLE
test: Test published points contract

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* https://github.com/Energinet-DataHub/opengeh-wholesale/issues/32

### CODE REVIEW

File: source/contracts/published-time-series-points.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+             "name": "TransactionId", 
+             "type": "string"
+        },
+        {
+             "name": "Quantity", 
+             "type": "decimal(18,3)"
+        },
+        {
+             "name": "Quality", 
+             "type": "long"
```
Comment (defectiveAi) [inline source/contracts/published-time-series-points.json:17]: Why 64-bit for Quality and Resolution. Is it not a regular 32-bit enum value?

File: source/databricks/tests/contract_utils.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    # Assert: The enum values must match contract
+    for literal in literals:
+        supported_arg = next(x for x in supported_literals if literal.name == x["name"])
+        assert literal.value == supported_arg["value"]
+
+
+def get_message_type(contract_path):
+    grid_area_updated_schema = read_contract(contract_path)
+    return next(
+        x for x in grid_area_updated_schema["bodyFields"] if x["name"] == "MessageType"
```
Comment (defectiveAi) [inline source/databricks/tests/contract_utils.py:60]: I do not think this method is in use. The name is 'grid_area_updated_schema' and the index is 'bodyFields' and not 'fields'.

Comment (BjarkeMeier) [reply source/databricks/tests/contract_utils.py]: Thx. Fixed.

**Review by defectiveAi — APPROVED** [review]

**Review by johevemi — APPROVED** [review]

File: source/contracts/published-time-series-points.json (hunk 1 above)
Comment (BjarkeMeier) [reply source/contracts/published-time-series-points.json:17]: It's a reminiscence from Python 2. In Python 3 long and int are the same. I'll update the types in another PR.

**Pull request merged**